api/v1alpha1/drplacementcontrol_types.go

```diff
@@ -104,7 +119,15 @@ const (
 // DRPlacementControlSpec defines the desired state of DRPlacementControl
 type DRPlacementControlSpec struct {
 	// PlacementRef is the reference to the PlacementRule used by DRPC
-	PlacementRef v1.ObjectReference `json:"placementRef"`
+	PlacementRef v1.ObjectReference `json:"placementRef,omitempty"`
```

Comment:
This change makes it no longer a required field.  We need to make sure that we validate the DRPC CR early that the PlacementRef is provided when `OCMDiscovered` apps are not used. Or should we just leave it as `required` field but provide a no-op PlacementRef?

Reply:
Good question. I am leaning towards the first approach and therefore the changes in the drpc reconcile function. If I read the ramen config map early and also inspect the drpc spec, I will be able to determine whether the drpc is a OCM managed one or OCM discovered one. This way there both managed and discovered apps are treated the same. In the second approach, discovered apps feel like an afterthought.